### PR TITLE
Cut 0.20.0

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+
+1. Automated live migration: If the Prometheus Operator finds Kubernetes
+   ConfigMaps that match the `RuleSelector` in a `Prometheus` specification, it
+   will convert them to matching `PrometheusRule` resources.
+
+2. Manual migration: We provide a basic CLI tool to convert Kubernetes
+   ConfigMaps to `PrometheusRule` resources.
+
+```bash
+go get -u github.com/coreos/prometheus-operator/cmd/po-rule-cm-to-rule-file-crds
```
Comment (brancz) [inline CHANGELOG.md]: should we be renaming this? `po-rule-cm-to-prometheus-rule-crds`?

Comment (ant31) [reply CHANGELOG.md]: po-rules-migration

Comment (mxinden) [reply CHANGELOG.md]: Works for me. @brancz would you mind addressing this in https://github.com/coreos/prometheus-operator/pull/1430 ?

**Comment by mxinden** [comment]
Updated the changelog to `PrometheusRule` rename patch.

@brancz @ant31 Let me know if you have further comments.

**Pull request merged**